pxb/v2/local/test-binary

```diff
@@ -93,8 +93,10 @@ else
     fi
     ./bootstrap.sh --pxb-type=${CMAKE_BUILD_TYPE} --type=${XTRABACKUP_TARGET} --version=${XTRABACKUP_TARGET_VERSION}
 fi
-if [[ ${CMAKE_BUILD_TYPE} == "Debug" ]] && [[ ${XTRABACKUP_TARGET} == "innodb80" ]]; then
-    ./run.sh ${XBTR_ARGS} -f -D | tee -a ${WORKDIR_ABS}/xbtr.output
+if [[ ${CMAKE_BUILD_TYPE} == "Debug" ]]; then
+    if [[ ${XTRABACKUP_TARGET} == "innodb80" ]] || [[ ${XTRABACKUP_TARGET} == "innodb57" ]]; then
```

Comment:
It is not, because we have xtradb80 target as well, in our case we use separate tarball, without mysqld-debug binary, symbols are compiled into mysqld binary
in Oracle's targets it's a separate binary and we use `-D` flag to use it